translations/ES_eu (international Spanish really)

```diff
@@ -0,0 +1,128 @@
+---
+en:
+  title: "Bitcoin Cash - Dinero electrónico de usuario a usuario"
+  meta:
+    description: |
+      Bitcoin Cash es una moneda sólida para todos. Usuarios y comerciantes pueden ahora disfrutar de comisiones bajas y confirmaciones fiables. El futuro es prometedor gracias a crecimiento sin restricciones, adopción global, innovación que no necesita permisos y desarrollo descentralizado.
+    twitter:
+      title: "Bitcoin Cash - Dinero electrónico de usuario a usuario"
+      description: |
+        Bitcoin Cash es una moneda sólida para todos. Usuarios y comerciantes pueden ahora disfrutar de comisiones bajas y confirmaciones fiables. El futuro es prometedor gracias a crecimiento sin restricciones, adopción global, innovación que no necesita permisos y desarrollo descentralizado.
+    og:
+      title: "Bitcoin Cash - Peer-to-Peer Electronic Cash"
+      description: |
+        Bitcoin Cash es una moneda sólida para todos. Usuarios y comerciantes pueden ahora disfrutar de comisiones bajas y confirmaciones fiables. El futuro es prometedor gracias a crecimiento sin restricciones, adopción global, innovación que no necesita permisos y desarrollo descentralizado.
+    misc:
+      logo: "logo"
+    sections:
+      about: "Acerca de"
+      features: "Características"
+      nodes: "Nodos"
+      wallets: "Monederos"
+      community: "Comunidad"
+      services: "Servicios"
+      projects: "Proyectos"
+      exchanges: "Casas de cambio"
+      faq: "FAQ"
+    layout:
+      bitcoin_cash: "Bitcoin Cash"
+      lead: "Dinero electrónico de usuario a usuario"
+      loading: "Cargando..."
+    features:
+      - title: "rápido"
+        description: "Transacciones en segundos, confirmaciones en minutos."
+      - title: "Fiable"
+        description: "Una red sin congestión"
+      - title: "Low Fees"
+        description: "Envía dinero a cualquier parte por céntimos."
+      - title: "Simple"
+        description: "Fácil de usar y sin complicaciones."
+      - title: "Estable"
+        description: "Un sistema de pago que demuestra ser una reserva de valor."
+      - title: "Seguro"
+        description: "La tecnología blockchain más robusta del mundo."
+    headings:
+      about: "El mejor dinero del mundo"
+      features: "Nuevas características"
+      nodes: "Nodos"
+      wallets: "Monederos"
+      services: "Servicios"
+      projects: "Proyectos"
+      exchanges: "Casas de cambio"
+      faq: "Preguntas frecuentes"
+    content:
+      about:
+        - |
+          Bitcoin Cash ofrece una moneda sólida al mundo, cumpliendo con la promesa inicial de Bitcoin como "Dinero electrónico Peer-to-Peer" (de usuario a usuario). Usuarios y comerciantes pueden disfrutar de comisiones bajas y confirmaciones fiables. El futuro es prometedor gracias a crecimiento sin restricciones, adopción global, innovación que no necesita permisos y desarrollo descentralizado.
```

Comment:
That's fair, I agree it is inconsistent but I opted to include the original term here as the sentence refers back to Satoshi Nakamoto's creation (plus peer to peer / P2P is a relatively known term in the internet world since torrent downloads came to life). We can remove it for consistency if you consider it doesn't fit there, np.

Reply:
No need to remove, makes sense. Also feel free to change "usuario a usuario" if you feel there's a better translation for peer-to-peer. Maybe "par a par" or similar.

Reply:
I think usuario a usuario is perfect! "par a par" o "entre pares" sounds strange to the mainstream public